channels/dk.m3u

```diff
@@ -11,8 +11,6 @@ http://stream.kkr.dk/live/ngrp:kkr_adaptive/chunklist.m3u8
 http://stream.kkr.dk/live/ngrp:kkr_adaptive/playlist.m3u8
 #EXTINF:-1 tvg-id="KVF.dk" tvg-name="KVF" tvg-country="DK" tvg-language="Danish" tvg-logo="https://i.imgur.com/Nb93Skv.jpg" group-title="",KVF (720p)
 https://w1.kringvarp.fo/uttanlands/_definst_/smil:uttanlands.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="KVF.dk" tvg-name="KVF" tvg-country="DK" tvg-language="Danish" tvg-logo="https://i.imgur.com/Nb93Skv.jpg" group-title="",KVF (480p)
-https://w2.kringvarp.fo/uttanlands/480/manifest.m3u8
```

Comment:
Not 24/7

Works, but sounds like it comes from an adaptive playlist that I do not possess.